scripts/classification/imagenet/verify_pretrained.py

```diff
@@ -59,9 +59,13 @@
 
 normalize = transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
 
+"""
+Aligning with TF implemenation, the crop-input ratio is 0.875
+Set the crop 342 for input size 299 accordingly. 
+"""
 if 'inceptionv3' in model_name:
     transform_test = transforms.Compose([
-        transforms.Resize(320, keep_ratio=True),
+        transforms.Resize(342, keep_ratio=True),
```

Comment:
it doesn't make sense to fix `342` while having `input_size` as a constant. They should match each other's value, i.e. either both are constant or both are input args.